### ISSUE TITLE
Add manual installation instructions to readme

### ISSUE THREAD

**Issue by shafy** [hotwired/stimulus-rails#94]
When running the installer with with `rails stimulus:install` on an existing project, it often fails to corretly update files to add the necessary lines of code, showing this error message: `File unchanged! The supplied flag value not found!`.

I think it would be helpful to add a section that explains the manual install to the readme, so when somebody installs it and sees that it failed to update a file correctly, they can just do it manually. In addition, we could add what it was supposed to do to the error message, something like:

```

File unchanged! The supplied flag value not found!  app/javascript/application.js

Wanted to add:
import "@hotwired/stimulus-loading"

to app/javascript/application.js, but couldn't parse the file correctly. Please add manually or retry.
```

Happy to look into this.

**Comment by dhh** [comment]
Please do investigate 👍

**Comment by shafy** [comment]
Ok, I've looked into it and it looks like the weird error thrown by Thor (`File unchanged! The supplied flag value not found!`) is an issue with Thor that's already being worked on ([see here](https://github.com/rails/thor/issues/706)).

Nevertheless, I created a PR that attempts to make the installation process more transparent for the user with [this PR](https://github.com/hotwired/stimulus-rails/pull/95).
